### PR TITLE
[EOPS-678] Modify v_invalid_concept_ids pg migration

### PR DESCRIPTION
**Pull request by na2960**
- Modifies the invalid_concept_ids query

### CODE REVIEW

**Review by jp3477 — APPROVED** [review]
This looks good to me!

File: sql/ehr_metrics/invalid_concept_ids/invalid_concept_ids.sql [hunk 1]
```diff
@@ -1,28 +1,28 @@
 SELECT
-  "visit_occurrence" AS table_name,
-  "visit_concept_id" AS column_name,
+  'visit_occurrence_id' AS table_name,
```
Comment (xj2193) [inline sql/ehr_metrics/invalid_concept_ids/invalid_concept_ids.sql:2]: We want to keep 'visit_occurrence' as the table name.

**Review by xj2193 — CHANGES_REQUESTED** [review]
Thanks @na2960 , the only part needs to be changed are the table_names column. Could you revert them back to the omop table names. This field is for indicating which OMOP table has invalid concept id.

**Comment by na2960** [comment]
> Thanks @na2960 , the only part needs to be changed are the table_names column. Could you revert them back to the omop table names. This field is for indicating which OMOP table has invalid concept id.

Oh yes, I forgot to revert this back. Should be updated now w/ the last commit. Thanks Zoey!

**Comment by na2960** [comment]
Updated table_name strings to reflect table itself (vs ...._id)

**Comment by jp3477** [comment]
Sorry, accidentally closed it! It's reopened now.

**Pull request closed without merging**